### ISSUE TITLE
ensure all amheida PDFs are accessible before publication

### ISSUE THREAD

**Issue by paregorios** [isawnyu/isaw.web#409]
_governing epic: #408 migrate amheida content from standalone website to the research section of isaw web_

General experience suggests that some or all of the PDFs included in the Amheida section are likely not prepared properly for accessibility. The latter condition is a WCAG violation and must be corrected before the Amheida materials can be published.

Steps:

 - [ ] determine which pdfs need remediation (make a list)
 - [ ] ask Amheida team to re-generate as many of these as possible from the original source, using accessibility methods
 - [ ] verify regenerated PDFs meet accessibility guidelines and swap them out through the plone
 - [ ] arrange mediation of PDFs that cannot be re-generated from source and post the results

**Comment by paregorios** [comment]
Assigned this to myself to do the first step: make a list of all amheida-related images and whether they need alt text or not.
